src/main/java/com/couchbase/lite/BlobStore.java

```diff
@@ -555,30 +554,33 @@ public boolean storeBlob(byte[] data, BlobKey outKey) {
     }
 
     private static byte[] getBytesFromFile(File file) throws IOException {
-        InputStream is = new FileInputStream(file);
+        InputStream is = null;
+        try {
+            is = new FileInputStream(file);
 
-        // Get the size of the file
-        long length = file.length();
+            // Get the size of the file
+            long length = file.length();
 
-        // Create the byte array to hold the data
-        byte[] bytes = new byte[(int) length];
+            // Create the byte array to hold the data
+            byte[] bytes = new byte[(int) length];
```

Comment:
This method loads entire content into the memory from the file. If a file is large, `new byte[length]` fails and return null. Should check return value from `new byte[length]`.